cmd/backup/script.go

```diff
@@ -89,6 +89,18 @@ func newScript() (*script, error) {
 		return nil
 	})
 
+	envconfig.Lookup = func(key string) (string, bool) {
+		location, ok := os.LookupEnv(key + "_FILE")
```

Comment:
I see the probability of people having a random, stray `_FILE` variant floating around (specifically for the backup container) as very very low.

I checked out the `postgres` Docker image source ([entrypoint](https://github.com/docker-library/postgres/blob/master/16/alpine3.18/docker-entrypoint.sh)) and found this:

```sh
file_env() {
	local var="$1"
	local fileVar="${var}_FILE"
	local def="${2:-}"
	if [ "${!var:-}" ] && [ "${!fileVar:-}" ]; then
		printf >&2 'error: both %s and %s are set (but are exclusive)\n' "$var" "$fileVar"
		exit 1
	fi
	local val="$def"
	if [ "${!var:-}" ]; then
		val="${!var}"
	elif [ "${!fileVar:-}" ]; then
		val="$(< "${!fileVar}")"
	fi
	export "$var"="$val"
	unset "$fileVar"
}
```

TLDR:
- If both variants are present -> panic
- Prefer the value over the _FILE variant

Makes sense to me, partially because that's the `old` way of doing it and you can see the contents of the variable right in the configuration file, not indirectly in e.g. Docker secrets.

Reply:
Sounds good to me as well, yes.